app/src/ui/repositories-list/repository-list-item.tsx

```diff
@@ -71,33 +70,14 @@ export class RepositoryListItem extends React.Component<
       prefix = `${gitHubRepo.owner.login}/`
     }
 
-    const className = enableGroupRepositoriesByOwner()
-      ? 'repository-list-item group-repositories-by-owner'
-      : 'repository-list-item'
+    const className = 'repository-list-item group-repositories-by-owner'
```

Comment:
I think we should take this opportunity to delete the `:not(.group-repositories-by-owner)` meta selector from `_repository_list.scss` since that's never going to be applied now. If you want to tackle that in a separate PR that's fine but I think that a part of removing the feature flags should be removing these feature flag specific css classes.

Reply:
I agree and thanks for the pointer! since these changes didn't remove any CSS class I thought there would not be any styles that we could remove (didn't think about `:not()` lol).